matchms/filtering/remove_spectra_within_tolerance.py

```diff
@@ -0,0 +1,35 @@
+import numpy
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def remove_spectra_within_tolerance(spectrum: SpectrumType, mz_tolerance: float = 17) -> SpectrumType:
```

Comment:
```suggestion
def remove_peaks_around_precursor_mz(spectrum: SpectrumType, mz_tolerance: float = 17) -> SpectrumType:
```

Reply:
It's removing peaks not spectra and "within tolerance" seemed to unspecific?

Reply:
Yeah should've been peaks and not spectra. "around" is more clear than just tolerance, so I think that's fine.